tensorboard/webapp/runs/views/runs_table/runs_data_table_test.ts

```diff
@@ -150,19 +151,58 @@ describe('runs_data_table', () => {
     expect(cells[3].componentInstance.header.name).toEqual('color');
   });
 
-  it('disables controls for color header', () => {
-    const fixture = createComponent({});
+  describe('color column', () => {
+    it('disables controls for color header', () => {
+      const fixture = createComponent({});
 
-    const dataTable = fixture.debugElement.query(
-      By.directive(DataTableComponent)
-    );
-    const headers = dataTable.queryAll(By.directive(HeaderCellComponent));
+      const dataTable = fixture.debugElement.query(
+        By.directive(DataTableComponent)
+      );
+      const headers = dataTable.queryAll(By.directive(HeaderCellComponent));
 
-    const colorHeader = headers.find(
-      (h) => h.componentInstance.header.name === 'color'
-    )!;
+      const colorHeader = headers.find(
+        (h) => h.componentInstance.header.name === 'color'
+      )!;
+
+      expect(colorHeader.componentInstance.controlsEnabled).toBe(false);
+    });
+
+    it('renders group by control in color header', () => {
+      const fixture = createComponent({});
+
+      const dataTable = fixture.debugElement.query(
+        By.directive(DataTableComponent)
+      );
+      const headers = dataTable.queryAll(By.directive(HeaderCellComponent));
 
-    expect(colorHeader.componentInstance.controlsEnabled).toBe(false);
+      const colorHeader = headers.find(
+        (h) => h.componentInstance.header.name === 'color'
+      )!;
+
+      expect(colorHeader.query(By.css('runs-group-menu-button'))).toBeTruthy();
+    });
+
+    it('renders color picker button in color content cells', () => {
```

Comment:
This test is bad. I can copy over the TestableColorPicker and ColorPickerTestHelper stuff from the runs_table_test.

Reply:
I think this test is fine, but it would be nice to have another test that checks the color picker opens and stays open.